versioned-lambda/orb.yml

```diff
@@ -8,7 +8,7 @@ orbs:
 executors:
   lambci-node12:
     docker:
-      - image: lambci/lambda:build-nodejs12.x
```

Comment:
what's the long term fix for this?

Reply:
We'll have to look at running the snyk/scan step in a different container that doesn't have the issue mentioned below. Don't suppose you have any recommendations for other containers?

Reply:
Or potentially run the entire node-test-and-package job with a different container